admin/package.json

```diff
@@ -39,7 +39,7 @@
     "**/**/ajv": "^6.12.6"
   },
   "dependencies": {
-    "@azure/service-bus": "^1.1.0",
+    "@azure/service-bus": "^7.0.5",
```

Comment:
I didn't expect us to upgrade this for `admin` until July at the earliest. One alternative would be to move this to a HTTP function to interface with queues.

Reply:
Yes i considered this, and we use it in both tslib and admin.  It has a smaller footprint on the admin app, and is used only to put prepare-check messages on the queue.

There are advantages of using version 7 in comparison to 1 for this PR, and there is nothing compelling against not using it, as the pin generation journey is tested on each PR.